### PR TITLE
fix(sdk-core): eth supports tss

### PR DESCRIPTION
**Pull request by olibeyene**
Ticket: BG-53272

### CODE REVIEW

**Review by wesgraham — APPROVED** [review]

File: modules/bitgo/test/v2/unit/staking/stakingWalletNonTSS.ts [hunk 1]
```diff
@@ -24,12 +24,12 @@ describe('non-TSS Staking Wallet', function () {
   before(function () {
     bitgo = TestBitGo.decorate(BitGo, { env: 'mock', microservicesUri } as any);
     bitgo.initializeTestVars();
-    baseCoin = bitgo.coin('eth');
```
[the remark this answers is not in the document]
Comment (olibeyene) [reply modules/bitgo/test/v2/unit/staking/stakingWalletNonTSS.ts:27]: I will be changing this back to eth and add logic in staking to make eth work as is.

File: modules/sdk-core/src/bitgo/staking/stakingWallet.ts [hunk 2]
```diff
@@ -90,7 +90,7 @@ export class StakingWallet implements IStakingWallet {
    * @param transaction - staking transaction to build
    */
   async build(transaction: StakingTransaction): Promise<StakingPrebuildTransactionResult> {
-    if (this.wallet.baseCoin.supportsTss()) {
+    if (this.wallet.baseCoin.supportsTss() && this.wallet.baseCoin.getFamily() !== 'eth') {
```
Comment (mmcshinsky-bitgo) [inline modules/sdk-core/src/bitgo/staking/stakingWallet.ts:93]: Can we document not supporting eth family in the jsdocs?

File: modules/sdk-core/src/bitgo/wallet/wallet.ts [hunk 3]
```diff
@@ -79,7 +80,7 @@ export class Wallet implements IWallet {
   public readonly bitgo: BitGoBase;
   public readonly baseCoin: IBaseCoin;
   private _wallet: WalletData;
-  private readonly tssUtils: ITssUtils;
+  private readonly tssUtils: EcdsaUtils | EddsaUtils | undefined;
```
Comment (mmcshinsky-bitgo) [inline modules/sdk-core/src/bitgo/wallet/wallet.ts:83]: For educational purposes, what's the logic in this change?

Comment (olibeyene) [reply modules/sdk-core/src/bitgo/wallet/wallet.ts:83]: ITssUtils is depracted. ITssUtils = EddsaUtils however tss now also supports EcdsaUtils

File: modules/sdk-core/src/bitgo/staking/stakingWallet.ts (hunk 2 above)
Comment (olibeyene) [reply modules/sdk-core/src/bitgo/staking/stakingWallet.ts:93]: staking team is aware of this change. This change is requires to keep eth as is today while we add support tss for eth. Denny Becker is going to put it on their road map to change this.

**Review by mmcshinsky-bitgo — APPROVED** [review]

**Pull request merged**